jupyter_client/session.py

```diff
@@ -199,14 +198,14 @@ class Message(object):
 
     def __init__(self, msg_dict):
         dct = self.__dict__
-        for k, v in iteritems(dict(msg_dict)):
+        for k, v in dict(msg_dict).items():
```

Comment:
i'm not sure why `ipython_genutils.py3compat` refers to `iteritems` as `iter(dict.items())` when on python 3 `dict.items()` already returns an iterator!